src/chainstate/burn/operations/mod.rs

```diff
@@ -42,6 +42,7 @@ use crate::chainstate::stacks::address::PoxAddress;
 use crate::util_lib::db::DBConn;
 use crate::util_lib::db::DBTx;
 use crate::util_lib::db::Error as db_error;
+use serde::Deserialize;
```

Comment:
Nit: I'd prefer to put this import between the std import and the crate imports.

Reply:
Moved in 92ec39e4d36420fb9b5c966ac549a84ffc3417cc